capture/moloch.h

```diff
@@ -571,6 +572,8 @@ typedef struct molochpacket_t
     uint8_t        ipProtocol;     // ip protocol
     uint8_t        mProtocol;      // moloch protocol
     uint8_t        readerPos;      // position for filename/ops
+    uint8_t        ethCount;       // the number of ethernet frames found
```

Comment:
this could be a :3 if you change the max to 8. make sure you move to other bit fields